easel/lib/screens/creator_hub/creator_hub_screen.dart

```diff
@@ -198,19 +207,37 @@ class _CreatorHubContentState extends State<CreatorHubContent> {
                   padding: EdgeInsets.symmetric(horizontal: 20.w),
                   child: Row(
                     children: [
-                      buildOutlinedBox(title: LocaleKeys.draft.tr(), viewModel: viewModel, collectionType: CollectionType.draft),
+                      buildOutlinedBox(
+                        title: LocaleKeys.draft.tr(),
+                        viewModel: viewModel,
+                        collectionType: CollectionType.draft,
+                      ),
                       SizedBox(width: 14.w),
-                      buildOutlinedBox(title: LocaleKeys.published.tr(), viewModel: viewModel, collectionType: CollectionType.published),
+                      buildOutlinedBox(
+                        title: LocaleKeys.published.tr(),
+                        viewModel: viewModel,
+                        collectionType: CollectionType.published,
+                      ),
                       const Expanded(child: SizedBox()),
                       SizedBox(width: 16.w),
                       InkWell(
                         onTap: () => viewModel.updateViewType(ViewType.viewGrid),
-                        child: SvgPicture.asset(SVGUtils.kGridIcon, height: 15.h, color: viewModel.viewType == ViewType.viewGrid ? EaselAppTheme.kBlack : EaselAppTheme.kGreyIcon),
+                        child: SvgPicture.asset(
+                          SVGUtils.kGridIcon,
+                          height: 15.h,
+                          color:
+                              viewModel.viewType == ViewType.viewGrid ? EaselAppTheme.kBlack : EaselAppTheme.kGreyIcon,
```

Comment:
why its in new line? I think I should be in the same line.

Reply:
Same here.